include/dlaf/blas/tile.h

```diff
@@ -113,9 +113,95 @@ struct CublasGemm<std::complex<double>> {
   }
 };
 
+template <typename T>
+struct CublasHemm;
+
+template <>
+struct CublasHemm<float> {
+  template <typename... Args>
+  static void call(Args&&... args) {
+    DLAF_CUBLAS_CALL(cublasSsymm(std::forward<Args>(args)...));
+  }
+};
```

Comment:
In your fight against repetitions, I would suggest you to think about using macros to cleanup this too.

e.g.
```
DLAF_CREATE_CUBLAS_OP(Hemm);
DLAF_CREATE_CUBLAS_OP(Hemm, float,                cublasSsymm);
DLAF_CREATE_CUBLAS_OP(Hemm, double,               cublasDsymm);
DLAF_CREATE_CUBLAS_OP(Hemm, std::complex<float>,  cublasChemm);
DLAF_CREATE_CUBLAS_OP(Hemm, std::complex<double>, cublasZhemm);
```

Reply:
Is what I did with Gemm ok?

Reply:
Yep, I think it is way cleaner!

Just wondering if it is the right place for the macro definition, or it would be better to put them in `util_cublas` together with the `DLAF_CUBLAS_CALL`. Up to you 😉